### ISSUE TITLE
Please provide sdkman-init command.

### ISSUE THREAD

**Issue by mike-neck** [sdkman/sdkman-cli#354]
Congrats updating sdkman from gvm.

By the way, after updating from gvm, sdk command is not recognized by new another terminal.

The cause might be that `gvm selfupdate force` doesn't providing code for running `sdkman-init.sh`.

Could you provide the following code to `.bash_profile`?

```
export SDKMAN_DIR="/Users/name/.sdkman"
[[ -s "/path/to/user/.sdkman/bin/sdkman-init.sh" && -z $(which sdkman-init.sh | grep '/sdkman-init.sh') ]] && source "/path/to/user/.sdkman/bin/sdkman-init.sh"
```

**Comment by marcoVermeulen** [comment]
This should be fixed now, it was due to the fact that OSX provided a non-standard version of sed. It's been released as 3.0.1.

**Comment by mike-neck** [comment]
:+1:

**Comment by marcoVermeulen** [comment]
Could I please ask you to change the [page](http://mike-neck.hatenadiary.com/entry/2015/09/17/234228) that you have published regarding _how to updated sdkman from gvm_, as it is causing much confusion. None of these points are necessary as they were fixed very soon after the release (v3.0.2).
